src/lib/SX1280Driver/SX1280.cpp

```diff
@@ -80,14 +80,27 @@ bool SX1280Driver::Begin()
     return true;
 }
 
-void SX1280Driver::Config(SX1280_RadioLoRaBandwidths_t bw, SX1280_RadioLoRaSpreadingFactors_t sf, SX1280_RadioLoRaCodingRates_t cr, uint32_t freq, uint8_t PreambleLength, bool InvertIQ, uint8_t PayloadLength)
+void SX1280Driver::Config(SX1280_RadioLoRaBandwidths_t bw, SX1280_RadioLoRaSpreadingFactors_t sf, SX1280_RadioLoRaCodingRates_t cr, uint32_t freq, uint8_t PreambleLength, bool InvertIQ, uint8_t PayloadLength, uint32_t interval)
 {
     PayloadLength = PayloadLength;
     IQinverted = InvertIQ;
     SetMode(SX1280_MODE_STDBY_XOSC);
     ConfigLoRaModParams(bw, sf, cr);
     SetPacketParams(PreambleLength, SX1280_LORA_PACKET_IMPLICIT, PayloadLength, SX1280_LORA_CRC_OFF, (SX1280_RadioLoRaIQModes_t)((uint8_t)!IQinverted << 6)); // TODO don't make static etc. LORA_IQ_STD = 0x40, LORA_IQ_INVERTED = 0x00
     SetFrequencyReg(freq);
+    SetRxTimeoutUs(interval);
+}
+
+void SX1280Driver::SetRxTimeoutUs(uint32_t interval)
+{
+    if (interval)
+    {
+        timeout = interval * 1000 / 15625; // number of ticks for the sx1280 to timeout
```

Comment:
where is the `15625` coming from?

Reply:
I added some defines and comments to the top of the file.